### DISCUSSION TITLE
Openvslam documentation

### DISCUSSION THREAD

**Discussion by mirellameelo** [stella-cv/stella_vslam#111] [Q&A] [answered]
Hi!
I'm looking for more detailed Openvslam documentation. Usually, a work like that has more extensive documentation, maybe a dissertation, or thesis...  Something more detailed than the [paper](https://dl.acm.org/doi/pdf/10.1145/3343031.3350539) of 4 pages. If you have it, I'd be really grateful if you could share it. :)

**Comment by ymd-stella** [comment, marked as answer]
It doesn't exist.

**Reply by ymd-stella** [reply]
What kind of information do you need? Can we create a new one?

**Reply by ymd-stella** [reply]
(I found a relevant benchmark. https://hal.archives-ouvertes.fr/hal-02970830/document )

**Reply by mirellameelo** [reply]
Thanks for replying! Actually, right now, I don't need specific information... Since I used OpenVSLAM in my post-graduation work, I need to write about it in detail. But, as we don't have it, I'll be using the flow available in the paper and I'll assume the "traditional" theory regarding each topic, e.g. BA. When I finish it I can share it here, then others can read, complement.... etc. 
![flow_openvslam](https://user-images.githubusercontent.com/34249507/120414567-07ef9680-c30f-11eb-9c1e-ae0931b0af51.png)
